ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -158,6 +158,14 @@ def pytest_addoption(parser):
         dest='flexy_env_file',
         help="Path to flexy environment file"
     )
+    parser.addoption(
```

Comment:
Is this an option that should be repeatable or do we think that we will only want to change one part of the CSV?

Reply:
@clacroix12 I thought about making it possible to accept multiple values to replace but couldn't find a solution and also thought it doesn't worth investing time as up until now, we needed to change only 1 component at a time

Reply:
We definitely don't need to enhance this in this PR if it isn't necessary. That being said I don't think it will be difficult. If we make this parameter repeatable, the value of `--csv-change` would just end up with a list of values like this:

```python
["replace_me::replaced", "replace_me_too::also_replaced", ...]
```

The values being propagated to our config could then just be placed in a list format, and eventually iterated over when we go to replace the contents of the CSV with our replacement values.

I may be over-simplifying here or missing some part of the process that may cause complications but it seems like we could take this up in an enhancement if we saw the need to change multiple parts of the CSV for one execution.

Reply:
Discussed this offline and we will implement this in the future if it will be needed